assets/webpack.config.js

```diff
@@ -55,7 +55,7 @@ module.exports = (env, options) => {
       }),
       new MiniCssExtractPlugin({ filename: "../css/app.css" }),
       new MonacoWebpackPlugin({
-        languages: ["markdown", "elixir", "xml", "json", "sql", "html", "css"],
+        languages: ["markdown", "elixir", "xml", "json", "sql", "html", "css", "javascript", "js"],
```

Comment:
Both `js` and `javascript` should work in markdown, here we just specify which languages to include and I think it's just `javascript`, no?

```suggestion
        languages: ["markdown", "elixir", "xml", "json", "sql", "html", "css", "javascript"],
```

Reply:
You're absolutely right! I even tested it before pushing this PR but accidentally left it in it. Apologies that was careless of me.

Reply:
I pushed up the fix.

Reply:
No worries!